quay_search.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python2
+
+# Copyright 2016 Joachim Wolff
+# Mail: wolffj@informatik.uni-freiburg.de
+# License: MIT
+#
+# Chair of Bioinformatics
+# Department of Computer Science
+# Faculty of Engineering
+# Albert-Ludwigs-University Freiburg im Breisgau
+
+import argparse
+import json
+import os
+import shutil
+
+import requests
+
+from whoosh.fields import *
+from whoosh.index import create_in
+from whoosh.qparser import QueryParser
+from whoosh.spelling import Corrector
+from whoosh import qparser
+
+class QuaySearch():
+    """Tool to search within a quay organization for listed software. Prints out search results with docker download command."""
+    def __init__(self, p_organization):
+        self._index = None
+        self._organization = p_organization
+    def __del__(self):
+        pass
+
+    def build_index(self):
+        """This functions creates a index to quickly examine the repositories of a given quay organization."""
+        # download all information about the repositories from the
+        # given organization in self._organization
+        url = 'https://quay.io/api/v1/repository'
+        parameters = {'public':'true','namespace': self._organization}
+        r = requests.get(url, headers={'Accept-encoding': 'gzip'}, params=parameters,
+                            timeout=12)
+
+        # delete old index data and create the dir "indexdir" new
+        if os.path.exists("indexdir"):
```

Comment:
This should be a file under a temp directory. Otherwise we polluting a lot of directories with these files/folders.
Have a look at `tempfile.gettempdir()`.